src/http/index.js

```diff
@@ -29,6 +29,24 @@ function hapiInfoToMultiaddr (info) {
   return toMultiaddr(uri)
 }
 
+async function serverCreator (serverAddrsArr, createServerFunc, hapiInfoToMultiaddr, ipfs) {
+  if (!serverAddrsArr.length) {
+    debug(Error('There are no addresses'))
+  }
+  // just in case the address is just string
+  let serversAddrs = [].concat(serverAddrsArr)
+  const processServer = async (serverInstance, createServerFunc, hapiInfoToMultiaddr, ipfs) => {
```

Comment:
Rename `serverInstance` => `addr`/`address`, remove `createServerFunc`, `hapiInfoToMultiaddr`, `ipfs` - these are all already in scope.